src/test/java/org/rulez/demokracia/pdengine/CastVoteTest.java

```diff
@@ -42,4 +42,14 @@ public void the_preferences_can_be_obtained_when_they_contain_choices() {
 		List<RankedChoice> preferences = castVote.getPreferences();
 		assertEquals(theCastVote, preferences);
 	}
+
+	@TestedFeature("Unimplemented")
```

Comment:
Please put the test annotations to the class, which in turn means that you should make this test's own test class.
I suggest UnimmplementedTests or CastVoteUnimplementedTests as  a name